visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts

```diff
@@ -214,9 +214,9 @@ export class CodeMapMouseEventService
 	}
 
 	private onLeftClick() {
-		this.threeSceneService.clearSelection()
 		this.isGrabbing = false
 		if (!this.hasMouseMoved(this.mouseOnLastClick)) {
```

Comment:
Do you mind adding a threshold to the `hasMouseMoved` function? Moving the mouse by 1px should maybe count as not moved at all. A threshold of 5px should be good enough for a better user experience.

Reply:
Sure ! Good idea, added a threshold for the right and the left button of 3 px. If its to less, we can raise it.